### DISCUSSION TITLE
Tags in Quarto?

### DISCUSSION THREAD

**Discussion by vinhdizzo** [quarto-dev/quarto-cli#2984] [Q&A] [answered]
Migrating from WordPress. Does Quarto have a concept of tags? Or is it just categories? Thanks.

**Comment by dragonstyle** [comment, marked as answer]
Right now categories are the closest equivalent.

**Reply by vinhdizzo** [reply]
Thanks for confirming.

As I migrate, I plan to include my old tags in the yaml section similar to categories, in case it gets implemented in the future. Would there be any harm (error) doing this?

**Reply by dragonstyle** [reply]
That should cause no trouble at all (and you're right that could make things easier should they become supported in the future).
